deploy/zfs-operator.yaml

```diff
@@ -2058,7 +2058,7 @@ spec:
       serviceAccount: openebs-zfs-controller-sa
       containers:
         - name: csi-resizer
-          image: k8s.gcr.io/sig-storage/csi-resizer:v1.2.0
```

Comment:
deploy/zfs-operator.yaml is autogenerated yaml. This change will go away once you do make manifests

Reply:
Done, good catch!